### PR TITLE
Dev bd 765 telemetry command

### PR DESCRIPTION
**Pull request by ilaipx**

### CODE REVIEW

File: lib/px/utils/pxtelemetry.lua [hunk 1]
```diff
@@ -0,0 +1,32 @@
+local px_common_utils = require('px.utils.pxcommonutils')
+local px_constants = require('px.utils.pxconstants')
+local hmac = require "resty.nettle.hmac"
+
+local M = {}
+
+function M.telemetry_check_header(px_config, px_client, px_headers, px_logger)
```
Comment (yaronpx) [inline lib/px/utils/pxtelemetry.lua]: why do you need to pass `px_client` and `px_config` as params? you can just require them

File: lib/px/utils/pxtelemetry.lua [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    local header_value = px_headers.get_header(px_constants.ENFORCER_TELEMETRY_HEADER)
+    if not header_value then
+        return
+    end
+    px_logger.debug('Received command to send enforcer telemetry')
+    header_value = ngx.decode_base64(header_value)
+    local split_header_value = string.split(header_value, ':')
+    if #split_header_value ~= 2 then
+        px_logger.debug('Malformed ' .. px_constants.ENFORCER_TELEMETRY_HEADER .. ' header: ' .. header_value)
```
Comment (yaronpx) [inline lib/px/utils/pxtelemetry.lua]: you should probably exit the function at this point

File: lib/px/utils/pxtimer.lua [hunk 3]
```diff
@@ -13,26 +13,9 @@ function M.application(px_configutraion_table)
 	local px_logger = require ("px.utils.pxlogger").load(px_config)
 	local px_constants = require("px.utils.pxconstants")
 	local buffer = require "px.utils.pxbuffer"
-	local px_commom_utils = require('px.utils.pxcommonutils')
 
 	local ngx_timer_at = ngx.timer.at
 
-	function send_initial_enforcer_telemetry()
```
Comment (yaronpx) [inline lib/px/utils/pxtimer.lua]: why did you remove this?

File: lib/px/utils/pxtelemetry.lua (hunk 1 above)
Comment (ilaipx) [reply lib/px/utils/pxtelemetry.lua]: It requires calling load() method with px_config, seemed like double work to me. wdyt?

File: lib/px/utils/pxtimer.lua (hunk 3 above)
Comment (ilaipx) [reply lib/px/utils/pxtimer.lua]: The feature states that we should stop sending enforcer telemetry on init, because in some states when there are many instances, it sends too many.

File: lib/px/utils/pxtelemetry.lua (hunk 1 above)
Comment (yaronpx) [reply lib/px/utils/pxtelemetry.lua]: if that's the case, then I guess not
